README.md

```diff
@@ -18,19 +18,26 @@ For more background on the importance of monitoring outliers and distributions i
 ## Table of Contents
 
 
-* [Installation and Usage](#installation-and-usage)
-* [Supported Algorithms](#supported-algorithms)
-  * [Outlier Detection](#outlier-detection)
-  * [Adversarial Detection](#adversarial-detection)
-  * [Drift Detection](#drift-detection)
-  * [Reference List](#reference-list)
-* [Datasets](#datasets)
-  * [Sequential Data and Time Series](#sequential-data-and-time-series)
-  * [Images](#images)
-  * [Tabular](#tabular)
-* [Models](#models)
-* [Integrations](#integrations)
-* [Citations](#citations)
+- [![Slack channel](http://seldondev.slack.com/messages/alibi)](#)
```

Comment:
Slack channel and table of contents should not be hyperlinked here, check https://github.com/ojcobb/alibi-detect/tree/online_detectors#table-of-contents to see mismatch.

Reply:
Also seems like this branch is not rebased from master since it also references an outdated version in the citations section (0.6.0 instead of 0.6.2 with the old date).

Reply:
Hm weird, I'm not sure why this happened. Even after merging master the TOC and slack links remained so I have now deleted them manually.